standing_committees_and_working_groups.md

```diff
@@ -0,0 +1,36 @@
+# Standing Committees and Working Groups
+
+## Overview
+
+In addition to the software work on Jupyter that is coordinated through the [Software Steering Council](software_steering_council.md) (SSC), much of the project’s work expands beyond software. Examples include code of conduct incident response, diversity and inclusion, operations, legal, fundraising, events, community, and marketing. Standing Committees and Working Groups carry out this non-software related work of the project by delegation from the Executive Board (EB).
+
+The primary difference between Standing Committees and Working Groups is that Standing Committees are intended to be permanent; they are only created and dissolved by a joint vote of the EB and SSC. In contrast, Working Groups can be created and dissolved by the EB acting alone.
```

Comment:
To me, a dash suggests a clarification or expansion of an idea, while a semicolon suggests a separation of ideas.
```suggestion
The primary difference between Standing Committees and Working Groups is that Standing Committees are intended to be permanent—they are only created and dissolved by a joint vote of the EB and SSC. In contrast, Working Groups can be created and dissolved by the EB acting alone.
```